google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ui/CustomizableComboBox.java

```diff
@@ -219,7 +219,8 @@ public void paint(Graphics graphics) {
     super.paint(graphics);
     if ((textField.isFocusOwner() || (getPopup() != null && getPopup().isPopupVisible()))) {
       if (isUsingDarculaUiFlavor()) {
-        DarculaUIUtil.paintFocusRing(graphics, 3, 3, getWidth() - 4, getHeight() - 4);
+        DarculaUIUtil.paintSearchFocusRing((Graphics2D) graphics,
```

Comment:
casting Graphics to Graphics2d is apparently always safe: http://stackoverflow.com/questions/179415/java2d-is-it-always-safe-to-cast-graphics-into-graphics2d